src/main/java/com/example/project_spring_boot/service/UserServiceImpl.java

```diff
@@ -21,15 +24,21 @@ public void setUserRepository(UserRepository userRepository) {
     }
 
     @Override
-    public Item getUser(Long id) {
-        // TODO Auto-generated method stub
-        return null;
+    public User getUser(Long id) {
+        return userRepository.findById(id).get();
```

Comment:
I keep getting this warning - "Optional value should only be accessed after calling isPresent()"

Instead of .get() 
I suggest using .orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "Bad request")) 

It seems to resolve the issue.